### PR TITLE
Added OpenGraph metatags to head

### PR DESCRIPTION
**Pull request by PBJ12295**
Added OpenGraph metatags to head

### CODE REVIEW

File: views/templates/master.blade.php [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
 
-        <title>
-            <?php
-            echo apply_filters(
-              'Municipio/pageTitle',
-              wp_title('|', false, 'right') . get_bloginfo('name')
-            );
-            ?>
-        </title>
+        <title><?php echo $title; ?></title>
```
Comment (joso9001) [inline views/templates/master.blade.php]: Should probably be `{{ $title }}` to make sure it's properly escaped!

(Or, if it already is escaped `{!! $title !!}` to make it Blade-y.)

File: views/templates/master.blade.php [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
         <meta name="msapplication-TileColor" content="#0069b4">
         <meta name="theme-color" content="#0069b4">
+        
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
+
+        <link rel="apple-touch-icon" sizes="152x152" href="{{ get_stylesheet_directory_uri() }}/assets/images/favicon/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="{{ get_stylesheet_directory_uri() }}/assets/images/favicon/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="{{ get_stylesheet_directory_uri() }}/assets/images/favicon/favicon-16x16.png">
+        <link rel="manifest" href="{{ get_stylesheet_directory_uri() }}/assets/docs/site.webmanifest">
+        <link rel="mask-icon" href="{{ get_stylesheet_directory_uri() }}/assets/images/favicon/safari-pinned-tab.svg" color="#0069b4">
```
Comment (joso9001) [inline views/templates/master.blade.php]: Should these changes be here?

Might break if we merge it before #39.

File: views/templates/master.blade.php [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
 <html <?php language_attributes(); ?>>
     <head>
+        <meta property="og:title" content="<?php echo $title; ?>" />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="{{ the_permalink() }}" />
+        <?php
+        
+            if(has_post_thumbnail())
+                echo '<meta property="og:image" content="' . get_the_post_thumbnail_url() . '" />';
+        ?>
```
Comment (joso9001) [inline views/templates/master.blade.php]: I suggest Blade-ifying this!

```
@if(has_post_thumbnail())
    <meta property="og:image" content="{{ get_the_post_thumbnail_url() }}" />
@endif
```

**Comment by rsj10736** [comment]
I agree with the changes suggested by @joso9001. It's important to keep each pull request isolated to the problem it's addressing.

**Comment by PBJ12295** [comment]
Agreed to both comments, has been updated.

File: views/templates/master.blade.php [hunk 4]
```diff
@@ -1,28 +1,21 @@
...
+        
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
 
-        <link rel="apple-touch-icon" sizes="152x152" href="{{ get_stylesheet_directory_uri() }}/apple-touch-icon.png">
-        <link rel="icon" type="image/png" sizes="32x32" href="{{ get_stylesheet_directory_uri() }}/favicon-32x32.png">
-        <link rel="icon" type="image/png" sizes="16x16" href="{{ get_stylesheet_directory_uri() }}/favicon-16x16.png">
-        <link rel="manifest" href="{{ get_stylesheet_directory_uri() }}/site.webmanifest">
-        <link rel="mask-icon" href="{{ get_stylesheet_directory_uri() }}/safari-pinned-tab.svg" color="#0069b4">
-        <meta name="msapplication-TileColor" content="#0069b4">
-        <meta name="theme-color" content="#0069b4">
```
Comment (joso9001) [inline views/templates/master.blade.php]: Great! No sure if these should be removed though!

Comment (joso9001) [reply views/templates/master.blade.php]: Yeah, these lines should not be removed.

**Comment by JockeTF** [comment]
I created #43 instead as a minimal version of this.

**Pull request closed without merging**